### PR TITLE
🐛  SAT: fix logs slicing

### PR DESCRIPTION
**Pull request by antixar**
## What
Every logs chunk can include more than one line

## How
split every chunk per line

### CODE REVIEW

File: airbyte-integrations/bases/source-acceptance-test/source_acceptance_test/utils/connector_runner.py [hunk 1]
```diff
@@ -115,24 +115,28 @@ def run(self, cmd, config=None, state=None, catalog=None, **kwargs) -> Iterable[
...
-            if has_exception or "Traceback (most recent call last)" in line:
-                has_exception = True
-            else:
-                yield line
+            while True:
+                # every chunk can include several lines
+                found = buffer.find(b"\n")
+                if found <= -1:
+                    break
+                line = buffer[:found].decode("utf-8")
```
Comment (grubberr) [inline airbyte-integrations/bases/source-acceptance-test/source_acceptance_test/utils/connector_runner.py:126]: line = buffer[:found+1].decode("utf-8")
?

**Review by grubberr — APPROVED** [review]

File: airbyte-integrations/bases/source-acceptance-test/source_acceptance_test/utils/connector_runner.py (hunk 1 above)
Comment (antixar) [reply airbyte-integrations/bases/source-acceptance-test/source_acceptance_test/utils/connector_runner.py]: added

**Pull request merged**